deluge/httpdownloader.py

```diff
@@ -31,13 +33,38 @@
 log = logging.getLogger(__name__)
 
 
-class HTTPDownloader(client.HTTPDownloader):
+class BodyHandler(HTTPClientParser):
+    def __init__(self, request, finished, length, agent):
+        HTTPClientParser.__init__(self, request, finished)
```

Comment:
Better to use super so no don't need to refer to parent class so:
```python
super(BodyHandler, self).__init__(request, finished)
```

Reply:
This one will need me to change to `class BodyHandler(HTTPClientParser, object):` as `HTTPClientParser` is old style object, and super needs a new style class.

Reply:
Ah! I don't see that being an issue though, just explain that it in commit message.